content/en/docs/setup/best-practices/cluster-large.md

```diff
@@ -124,3 +124,6 @@ components, including cluster-critical addons.
 The [cluster autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#readme)
 integrates with a number of cloud providers to help you run the right number of
 nodes for the level of resource demand in your cluster.
+
+The [addon resizer](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#readme)
```

Comment:
The link target can be "https://github.com/kubernetes/autoscaler/blob/master/addon-resizer/README.md" or "https://github.com/kubernetes/autoscaler/blob/master/addon-resizer/", I don't understand what the current value means.

Reply:
https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#readme jumps past the GitHub page pager to the README itself.

Reply:
Example of how the user agent viewport looks if you visit that hyperlink
![example rendering](https://user-images.githubusercontent.com/22591623/122578683-054daa80-d04c-11eb-9ca2-582473939ce2.png)